erpnext/accounts/doctype/bank_account/bank_account.py

```diff
@@ -31,3 +31,10 @@ def make_bank_account(doctype, docname):
 	doc.is_default = 1
 
 	return doc
+
+
+@frappe.whitelist()
+def get_bank_account(bank_account):
+	if bank_account:
```

Comment:
since this function takes only one argument, if statement unnecessary. we're already checking if bank account was entered in `payment_entry.js`

Reply:
Fixed!